zerver/tests/test_upload.py

```diff
@@ -1246,17 +1247,21 @@ def tearDown(self) -> None:
 
 class RealmLogoTest(UploadSerializeMixin, ZulipTestCase):
 
-    def test_multiple_upload_failure(self) -> None:
+    def _test_multiple_upload_failure(self, night: bool) -> None:
         """
         Attempting to upload two files should fail.
         """
         # Log in as admin
         self.login(self.example_email("iago"))
         with get_test_image_file('img.png') as fp1, \
                 get_test_image_file('img.png') as fp2:
-            result = self.client_post("/json/realm/logo", {'f1': fp1, 'f2': fp2})
+            result = self.client_post("/json/realm/logo", {'f1': fp1, 'f2': fp2, 'night': ujson.dumps(night)})
         self.assert_json_error(result, "You must upload exactly one logo.")
 
+    def test_multiple_logo_upload_failure(self) -> None:
+        night = False
+        self._test_multiple_upload_failure(night)
```

Comment:
I think these would be better written as:
```
      self._test_multiple_upload_failure(night=False)
      self._test_multiple_upload_failure(night=True)
```

So that we test both cases, and we don't have an unnecessary line for setting the `night` variable.

Reply:
But I should keep tests for two logos different or it must be single test for both.

Reply:
But I should keep tests for two logos different or it must be single test for both.

Reply:
There's no need to have a separate test function for each -- it'll just be clutter.  I don't think anyone would have trouble debugging a test failure in a test function that looked like my suggestion -- it's clear from the lines of code whether the day or night one failed.